pontoon/base/static/js/translate.js

```diff
@@ -168,10 +168,31 @@ var Pontoon = (function (my) {
       var self = this;
       $("#editor")[0].entity = entity;
 
+      /*
+       * Linkifies any traces of URLs present in a given string.
+       * Usage : string.linkify();
+       *
+       * Matches the URL Regex and parses the required matches.
+       * Can find more than one URL in the given string.
+       */
+      if(!String.linkify) {
+        String.prototype.linkify = function() {
+          // http://, https://, ftp://
+          var urlPattern = /\b(?:https?|ftp):\/\/[a-z0-9-+&@#\/%?=~_|!:,.;]*[a-z0-9-+&@#\/%=~_|]/gim;
+          // www. sans http:// or https://
+          var pseudoUrlPattern = /(^|[^\/])(www\.[\S]+(\b|$))/gim;
+
+          return this
+            .replace(urlPattern, '<mark class="placeable"><a href="$&">$&</a></mark>')
+            .replace(pseudoUrlPattern, '<mark class="placeable">$1<a href="http://$2">$2</a></mark>');
+        };
+      }
+
       // Metadata: comments, sources, keys
       $('#metadata').empty();
       if (entity.comment) {
-        self.appendMetaData('Comment', this.doNotRender(entity.comment));
+        entity.comment = entity.comment.linkify();
+        self.appendMetaData('Comment', entity.comment);
```

Comment:
We have a lot of comments that contain markup and we don't want to render them:
https://public.etherpad-mozilla.org/p/pontoon.comments

We only want to render the ones that contain links. To detect them, I suggest you compare entity.comment to the output of the linkify(). If they are the same, we don't have a link, so we should set comment to this.doNotRender(entity.comment). Else, we should set it to the linkified value.